.github/workflows/ci.yml

```diff
@@ -146,6 +146,30 @@ jobs:
         run: cargo check --target wasm32-unknown-unknown
         env:
           RUSTFLAGS: --cfg=web_sys_unstable_apis
+  
+  build-wasm-atomics:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    needs: build
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ubuntu-assets-cargo-build-wasm-nightly-${{ hashFiles('**/Cargo.toml') }}
+      - uses: dtolnay/rust-toolchain@nightly
+        with:
+          target: wasm32-unknown-unknown
```

Comment:
```suggestion
      - uses: dtolnay/rust-toolchain@master
        with:
          toolchain: ${{ env.NIGHTLY_TOOLCHAIN }}
          target: wasm32-unknown-unknown
```

Reply:
I'm inexperienced with GitHub CI. Is this just a style consistency change or does it have another effect?

Reply:
we sometimes need to override the nightly version used because it's broken, so we have a variable set at the top of the workflow so that changing it once does it everywhere